test/integration/test/philips_light_test.js

```diff
@@ -7,8 +7,9 @@ const fs = require('fs');
 
 var philipshue_server = require('../lib/philipsHue_server.js');
 var nupnp_server = require('../lib/nupnp_PhilipsHue.js');
-var config = new Config('./lib/config/foxbox.js');
-var header = new Config('./lib/config/header.js').get('header');
+var config = new Config('./test/integration/lib/config/foxbox.js');
+var header = new Config('./test/integration/lib/config/header.js')
+.get('header');
```

Comment:
Nit: I missed it in my preview review, but having `header.header` looks redundant. How about changing header.js with something like:
```js
module.exports = {
  'Content-Type' : 'application/json'
};
```